src/templates/resources/footer.json

```diff
@@ -189,19 +189,36 @@
             }
         ],
         "currentCountryLocalisedName": "Danmark",
-        "currentCountryFlagKey": "dk",
         "countries": [
+            {
+                "key": "gb",
+                "flagUrl": "ukFlagIconUrl",
+                "localisedName": "United Kingdom",
+                "siteUrl": "https://www.just-eat.co.uk"
+            },
             {
                 "key": "au",
                 "flagUrl": "australiaFlagIconUrl",
-                "localisedName": "Australien",
+                "localisedName": "Australia",
                 "siteUrl": "https://www.menulog.com.au"
             },
             {
-                "key": "br",
-                "flagUrl": "brasilFlagIconUrl",
-                "localisedName": "Brasilien",
-                "siteUrl": "https://www.ifood.com.br"
+                "key": "at",
+                "flagUrl": "austriaFlagIconUrl",
+                "localisedName": "Österreich",
```

Comment:
Is it intentional that all of these values are the same for each tenant? i.e. they are no longer localised?

Reply:
The new approach is to have all the country names to be in their original language with one exception for Bulgaria

Reply:
Because it is legacy and when searchweb releases vue version, it will be deprecated I haven't changed the structure of the resource file and just changed the data